app/views/shared/dialogs/_dialog_provision.html.haml

```diff
@@ -1,29 +1,33 @@
 - wf = @edit[:wf] if @edit && @edit[:wf]
 = render :partial => "layouts/flash_msg"
 
-:javascript
-  dialogFieldRefresh.unbindAllPreviousListeners();
+- if Settings.product.new_dialog_user_ui
+  = render :partial => "shared/dialogs/dialog_user", :locals => {:wf => wf}
 
-.row
-  .col-md-12.col-lg-12
-    #dialog_tabs
-      - auto_refreshable_field_indicies = build_auto_refreshable_field_indicies(wf)
+- else
+  :javascript
+    dialogFieldRefresh.unbindAllPreviousListeners();
```

Comment:
Can you still call this after removing it from [here](https://github.com/ManageIQ/manageiq-ui-classic/pull/2067/files#diff-5ff734253a30e3d74999b1b1a5a4b91dL4) ?

Reply:
Yeah, good question. I restored the file and moved the new logic into an angular service.